app/src/main/java/it/niedermann/nextcloud/deck/ui/helper/colorchooser/ColorChooser.java

```diff
@@ -62,14 +87,36 @@ private void initDefaultColors() {
         }
     }
 
+    private void initCustomColorChooser() {
+        // initializes a final image icon for the custom color chooser and, if already set,
+        // will set the icon color to the custom color
+        customColorChooser = new ImageView(getContext());
+        if (!hasCustomColor) {
```

Comment:
```suggestion
               // instead of the whole if statement
               WhateverTypeItIs color = hasCustomColor ? this.selectedColor : R.color.board_default_custom_color;
               customColorChooser.setImageDrawable(ViewUtil.getTintedImageView(this.context, R.drawable.circle_alpha_colorize_36dp, color));
```

Reply:
Good suggestions, I can re-work it to match what you're suggesting here.

Reply:
@randy-jr-olive please update your branch. As soon as the suggestion has been implemented, we can merge it.